src/components/Todo/Todo.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import { User } from '../User';
```

Comment:
```suggestion
import PropTypes from 'prop-types';
import { User } from '../User';
```
Пробелы оставлять не обязательно

Reply:
я для зручності
спершу бібліотеки

потім компоненти

потім стилі, малюнки

Reply:
ctrl+click по компоненту, и тебя перекинет на его импорт, но это скорее стилистический комментарий

Reply:
ну це да. Воно просто зручніше коли все розділене, а не коли в одній кучці всі імпорти. Особливо коли компонент розростається

Reply:
що зручніше?) на імпорти ніхто не дивиться, їх може бути 20-30-40 штук. Вова правильно написав, ти просто проскролюєш імпорти і дивишся сам код, а якщо тебе цікавить реалізація конкретної імпортованої функції, об'єкта чи компонента ти просто переходиш через ctrl+click до реалізації.
Не треба розмазувати імпорти